EvidenceApi.Tests/V1/Gateways/DocumentsApiGatewayTests.cs

```diff
@@ -57,6 +59,27 @@ public async Task CanGetAClaimWithValidParameters()
             result.Should().BeEquivalentTo(expectedClaim);
         }
 
+        [Test]
+        public async Task CanCreateUploadPolicyWithValidParameters()
+        {
+            Guid id = Guid.NewGuid();
+
+            var expectedS3UploadPolicy = JsonConvert.DeserializeObject<S3UploadPolicy>(_s3UploadPolicyResponse);
+
+            _messageHandler.SetupRequest(HttpMethod.Post, $"{_options.DocumentsApiUrl}api/v1/documents/{id}/upload_policies", async request =>
+            {
+                var json = await request.Content.ReadAsStringAsync().ConfigureAwait(true);
+                var body = JsonConvert.DeserializeObject<Guid>(json);
```

Comment:
Not sure what is going on here—AFAIK this request has no body, the ID is already matched in the URL that you provided on line 69.

Just checking that the token is correct should be enough here 👍